manager/assets/modext/widgets/fc/modx.panel.fcprofile.js

```diff
@@ -51,7 +47,7 @@ MODx.panel.FCProfile = function(config) {
                     ,value: config.record.name
                     ,listeners: {
                         'keyup': {scope:this,fn:function(f,e) {
-                            Ext.getCmp('modx-fcp-header').getEl().update(_('profile') + ': ' + Ext.util.Format.htmlEncode(f.getValue()));
+                            Ext.getCmp('modx-fcp-header').getEl().update(Ext.util.Format.htmlEncode(f.getValue()));
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(f.getValue()));`

Reply:
Are you sure about that? I just looked at your PR and there are different class variants, for example - https://github.com/modxcms/revolution/pull/14890/files?file-filters%5B%5D=.js#diff-4fa92d4eddfe372ac291e0e50ad38ea4R67

Reply:
yes I'm sure, if you can, please correct the code from my PR as well, seems like I missed few of these after I created the updateHeader function...